modules/earthengine-layers/src/earth-engine-layer.js

```diff
@@ -17,6 +18,7 @@ const defaultProps = {
   /*
   data: object,
   token: string,
+  eeObject: String || object,
```

Comment:
Add a checkbox in the tracker issue to implement defaultProps?

Reply:
I was just reading the deck.gl docs now to write those up